lib/dal/src/job/definition/dependent_values_update.rs

```diff
@@ -485,6 +505,12 @@ async fn execution_error_detail(
     ))
 }
 
+type PrototypeFunctionExecutionResult = (
```

Comment:
Would be even better as a struct (not a blocker). I'm guilty of it too but we should definitely just make a type when we start returning a 3-tuple or bigger

Reply:
Absolutely. I think we need to do that precisely. I can do it in my follow-up PR to fix the filters.